qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1376,8 +1376,8 @@ def add_wavefunction_store(self, blobs_list: List[Dict[str, Any]]):
 
         Returns
         -------
-        TYPE
-            Description
+        Dict with keys: blob_id and meta
```

Comment:
Should be `List[Dict[str, Any]]` for the type.

Reply:
changed it to Dict[str, Any] since the returned object is a dict with keys meta and blob_ids, description brought in the next line.